src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -77,11 +77,46 @@ public static class RSessionEvaluationCommands {
             return evaluation.EvaluateAsync(script, REvaluationKind.Json);
         }
 
+        public static Task InstallPackage(this IRSessionInteraction interaction, string name) {
```

Comment:
We have RSessionInteractionCommands.cs for extensions of IRSessionInteraction.  I know we have some more in this file that probably should have been moved already, but haven't.  Up to you.